packages/core/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @see https://github.com/GoogleChrome/web-vitals/blob/main/src/lib/generateUniqueID.ts
+ */
+export const generateUID = () =>
+  `${Date.now()}-${Math.floor(Math.random() * (9e12 - 1)) + 1e12}`
+
+export const shuffle = <T extends any[]>(arr: T): T => {
+  for (let i = arr.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1))
+    ;[arr[i], arr[j]] = [arr[j], arr[i]]
```

Comment:
`;` typo?

Reply:
added by prettier
prefer not to touch it, as the shuffle function works (as seen in the tests)

otherwise I would probably look to outsource this to eg lodash

Reply:
we have no black magics here 😄 anyway, let's just leave it there 👍